_docs_integrate/integration-example.md

```diff
@@ -79,7 +79,7 @@ To do so, execute `POST /api/v2/Attributes` with the following payload:
 }
 ```
 
-You can query the Connector's address via the route `/api/v2/Account/IdentityInfo`. If you are using the Demo Connector of this Tutorial, the address is `did:e:stage.enmeshed.eu:dids:54013af2a3bffe9dc8cb99`.
+You can query the Connector's address via the route `/api/v2/Account/IdentityInfo`. If you are using the Demo Connector of this Tutorial, the address is `did:e:pilot.enmeshed.eu:dids:9364c879fcc41c87c6ecbd`.
 
```

Comment:
there must not be an empty line between the note and {: .notice--info}

Reply:
good catch! seems to be broke for a while.